doc/source/train/examples/torch_data_prefetch_benchmark/auto_pipeline_for_host_to_device_data_transfer.rst

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+auto_pipeline_for_host_to_device_data_transfer
+==============================================
+
+.. literalinclude:: /../../python/ray/train/examples/torch_data_prefetch_benchmark/auto_pipeline_for_host_to_device_data_transfer.py
```

Comment:
Would it be possible to include both the rst file and the python file?

Reply:
Thanks @amogkam for your feedback! Here is a [snapshot](https://ray--22974.org.readthedocs.build/en/22974/train/examples/torch_data_prefetch_benchmark/benchmark_example.html) of the update doc after a few rounds trial and error. The table doesn't look very good. If you have any ideas on improving it, please let me know!